tests/AP_gtest.h

```diff
@@ -14,9 +15,17 @@ ::std::ostream& operator<<(::std::ostream& os, const class_name_& param) \
     return os << param.printable_member_; \
 }
 
+/*
+* Setting the environment variable below is used to abort() from 
+* system.cpp panic() instead of staying in an infinite loop
+*/
+
 #define AP_GTEST_MAIN() \
 int main(int argc, char *argv[]) \
 { \
+    setenv("SITL_PANIC_UNIT_TEST_EXIT", "1", 0); \
```

Comment:
we should probably set those with WAF in the coverage setting

Reply:
@khancyr 
Would you point me in the right direction with an example? I've tried playing with the waf script to do this but I can't get it to work using cfg.setenv() 

I want to be sure that the tests can be run locally as well as on the server.

Reply:
Switched to using WEAK symbols instead